crates/viewer/re_space_view_map/src/visualizers/geo_points.rs

```diff
@@ -23,12 +25,22 @@ struct GeoPointEntry {
 
     /// Color.
     color: egui::Color32,
+
+    /// The instance corresponding to this entry.
+    instance_path: InstancePath,
 }
 
-/// A map scene, with entries on the map to render.
+/// Visualizer for [`GeoPoints`].
 #[derive(Default)]
 pub struct GeoPointsVisualizer {
+    /// Objects to render.
     map_entries: Vec<GeoPointEntry>,
+
+    /// Indices into `map_entries` corresponding to a given entity.
+    entities: ahash::HashMap<EntityPathHash, Vec<usize>>,
+
+    /// Indices into `map_entries` corresponding to specific instances.
+    instances: ahash::HashMap<InstancePathHash, usize>,
```

Comment:
no need to hash a hash

```suggestion
    entities: nohash_hasher::IntMap<EntityPathHash, Vec<usize>>,

    /// Indices into `map_entries` corresponding to specific instances.
    instances: nohash_hasher::IntMap<InstancePathHash, usize>,
```

Reply:
As discussed in slack, only one of these can be IntMap, which it is now.